### PR TITLE
[bitnami/multus-cni] feat: :tada: Add chart

### PR DESCRIPTION
**Pull request by javsalgar**
This PR adds a helm chart for Multus-CNI. It is based on the upstream [installation instructions](https://github.com/k8snetworkplumbingwg/multus-cni/blob/v3.9.3/deployments/multus-daemonset-thick-plugin.yml) for branch 3.x. As this updates the CNI configuration, it needs to be root and privileged.

### Benefits

New product in our catalog

### Possible drawbacks

n/a

### Applicable issues

- fixes #

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/multus-cni/templates/daemonset.yaml [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+          command:
+            - multus-daemon
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.args "context" $) | nindent 12 }}
+          {{- else if .Values.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.args "context" $) | nindent 12 }}
+          {{- else }}
+          args:
+          args:
```
Comment (fmulero) [inline bitnami/multus-cni/templates/daemonset.yaml:114]: typo here

File: bitnami/multus-cni/templates/clusterrolebinding.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{{- if .Values.rbac.create -}}
+kind: ClusterRoleBinding
+apiVersion: {{ include "common.capabilities.rbac.apiVersion" . }}
+metadata:
+  name: {{ include "common.names.fullname.namespace" . }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: server
```
Comment (fmulero) [inline bitnami/multus-cni/templates/clusterrolebinding.yaml:8]: Typo? `component` label is not defined in the rest of the kubernetes objects

**Review by FraPazGal — DISMISSED** [review]
The testing side LGTM!

**Review by fmulero — CHANGES_REQUESTED** [review]
Thanks @javsalgar !!!, I only found some little typos. Great work

File: bitnami/multus-cni/templates/_helpers.tpl [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{{/*
+Return the proper Gitea image name
```
[the remark this answers is not in the document]
Comment (javsalgar) [reply bitnami/multus-cni/templates/_helpers.tpl:2]: Copy-paste

File: .vib/multus-cni/goss/goss.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# This application performs modifications at host-level, injecting new configuration files. Therefore the
+# container has root user and is privileged. In order to properly test wherebauts, we would need to access the host and
```
[the remark this answers is not in the document]
Comment (javsalgar) [reply .vib/multus-cni/goss/goss.yaml:2]: copy-paste

File: bitnami/multus-cni/templates/clusterrolebinding.yaml (hunk 2 above)
Comment (javsalgar) [reply bitnami/multus-cni/templates/clusterrolebinding.yaml:8]: Fixed

File: bitnami/multus-cni/templates/daemonset.yaml (hunk 1 above)
Comment (javsalgar) [reply bitnami/multus-cni/templates/daemonset.yaml:114]: Fixeh

**Review by fmulero — APPROVED** [review]
LGTM

**Pull request merged**